Sources/SwiftDocC/Model/Rendering/Content/RenderBlockContent.swift

```diff
@@ -72,122 +72,84 @@ public enum RenderBlockContent: Equatable {
-        
-        /// Creates a new aside style of the given kind.
-        init(_ kind: Aside.Kind) {
-            switch kind {
-            case .note: self = .note
-            case .important: self = .important
-            case .warning: self = .warning
-            case .experiment: self = .experiment
-            case .tip: self = .tip
-            case .attention: self = .attention
-            case .author: self = .author
-            case .authors: self = .authors
-            case .bug: self = .bug
-            case .complexity: self = .complexity
-            case .copyright: self = .copyright
-            case .date: self = .date
-            case .invariant: self = .invariant
-            case .mutatingVariant: self = .mutatingVariant
-            case .nonMutatingVariant: self = .nonMutatingVariant
-            case .postcondition: self = .postcondition
-            case .precondition: self = .precondition
-            case .remark: self = .remark
-            case .requires: self = .requires
-            case .since: self = .since
-            case .todo: self = .todo
-            case .version: self = .version
-            case .throws: self = .throws
-            }
-        }
-
-        /// Create a new aside style by looking up its display name. Returns nil if no aside style matches.
-        init?(displayName: String) {
-            let casesAndDisplayNames = AsideStyle.allCases.map { (kind: $0, displayName: $0.displayName() )}
-            guard let matchingCaseAndDisplayName = casesAndDisplayNames.first(where: { $0.displayName == displayName }) else {
-                return nil
+    /// A type the describes an aside style.
+    public struct AsideStyle: Codable, Equatable {
+        /// A type that represents the different kinds of supported asides.
+        public struct Kind: RawRepresentable, Codable {
+            /// An important aside.
+            public static let important = Kind(rawValue: "important")
+            /// A warning aside.
+            public static let warning = Kind(rawValue: "warning")
+            /// An experiment aside.
+            public static let experiment = Kind(rawValue: "experiment")
+            /// A tip aside.
+            public static let tip = Kind(rawValue: "tip")
+            /// A note aside.
+            public static let note = Kind(rawValue: "note")
+            
+            /// The underlying raw string value.
+            public let rawValue: String
+            
+            /// Creates an aside style kind with the specified raw value.
+            /// - Parameter rawValue: The string to use as the underlying raw value.
+            public init(rawValue: String) {
+                self.rawValue = rawValue
             }
-            self = matchingCaseAndDisplayName.kind
         }
-
-        /// Returns the style of aside to use when rendering.
+        
+        /// The heading text to use when rendering this style of aside.
+        public let displayName: String
```

Comment:
Should we make this a `var`? There doesn't seem to be a reason why it should be a constant (e.g., there's no internal behavior that relies on this being constant)